careers/technical-engineer.md

```diff
@@ -1,73 +1,99 @@
 # Technical Engineer Path
 
-The technical engineer focuses on individual contributorship and technical leadership
+The technical engineer focuses on individual contributorship and technical leadership. Their contributions level up the quality, effectiveness and efficiency of the entire organization. Whereas the [software engineer path](software-engineer.md) focused on mastering the fundamentals of software development using a defined set of criteria, the senior technical path focuses more on projects/efforts that have strategic value for the company while driving technical design and up-leveling their team. Their impact is no longer additive, it is multiplicative. As engineers progress down this path, examples of success will be less pre-defined and more dependent on the contextual needs of the business.
 
 ## Senior Engineer
 
-Success as a Senior Engineer is based on their ability to make the team and individuals on the team more effective. A Senior Engineer's technical impact should be focused on using technology to increase the likelihood that Packback succeeds as a business. A Senior Engineer at Packback should be focused on making a 10x impact on the team; which can only be done by empowering others, optimizing processes, and identifying the highest impact projects. They are proficient at project management.
+The senior engineer displays a mastery of the core competencies of software engineering. Strong technical knowledge and high productivity are table stakes at this level. Growth opportunities focus on developing impact, communication, and collaboration skills in preparation for the staff engineer role.
```

Comment:
I can't think of a better way to phrase this, so we should probably keep as is. Yet, having the growth opportunities oriented around getting promoted to the next role seems incorrect. 

1. There are growth opportunities in the current role that aren't about going to the next (Senior Engineer is a completely acceptable stopping point career wise, not everyone wants to go to Staff).
2. If growth is just about getting to the next level, we've not properly articulated the desired outcomes of growth.